lib/css-compiler.js

```diff
@@ -10,24 +9,45 @@ module.exports = {
   writeCss
 };
 
+/**
+ * @returns {Object} result
+ *    {Promise<string>} result.output - A promise which resolves to path of the output css file.
+ *    {Promise<string>} result.compilation - A promise which resolves when css is written to disk.
```

Comment:
Why do we have two different Promises, instead of one Promise that resolves when CSS is written with the output of the CSS file?

Reply:
Could it return a Promise that development mode doesn't wait for? I definitely do think that the output/compilation combination for the result of this function is confusing, would appreciate any clarification.